tests/core.test.ts

```diff
@@ -0,0 +1,377 @@
+import { MPPAlert, Severity, alertType } from '../src/core/alert';
```

Comment:
Move the functionality that deletes items from the data base to some function that receives the neede parameters to delete the needed item.

Reply:
done